src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -214,907 +214,908 @@ void GetPurchasedAir(EnergyPlusData &state)
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Total Heating Rate",
                             Constant::Units::W,
-                            PurchAir(PurchAirNum).TotHeatRate,
+                            PurchAir.TotHeatRate,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Sensible Cooling Rate",
                             Constant::Units::W,
-                            PurchAir(PurchAirNum).SenCoolRate,
+                            PurchAir.SenCoolRate,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Latent Cooling Rate",
                             Constant::Units::W,
-                            PurchAir(PurchAirNum).LatCoolRate,
+                            PurchAir.LatCoolRate,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Total Cooling Rate",
                             Constant::Units::W,
-                            PurchAir(PurchAirNum).TotCoolRate,
+                            PurchAir.TotCoolRate,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Average,
+                            PurchAir.Name);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Supply Air Total Heating Fuel Energy Rate",
+                            Constant::Units::W,
+                            PurchAir.TotHeatFuelRate,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Average,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Supply Air Total Cooling Fuel Energy Rate",
+                            Constant::Units::W,
+                            PurchAir.TotCoolFuelRate,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Average,
+                            PurchAir.Name);
```

Comment:
Added two energy consumption rate report variables for supply air: `Zone Ideal Loads Supply Air Total Heating Fuel Energy Rate` and `Zone Ideal Loads Supply Air Total Cooling Fuel Energy Rate`.